schoolopy/main.py

```diff
@@ -146,6 +146,26 @@ def get_user(self, user_id):
         """
         return User(self._get('users/%s' % user_id))
 
+    def create_user(self, user_id, user):
+        """
+        Create a new user.
+
+        :param user_id: ID of user you wish to create.
+        :param user: User object containing necessary fields.
+        :return: User object obtained from API.
+        """
+        return User(self._post('users/%s' % user_id, user.json))
+
+    def update_user(self, user_id, user):
+        """
+        Update an existing user.
+
+        :param user_id: ID of user you wish to update.
+        :param user: User object containing necessary fields.
+        :return: User object obtained from API.
+        """
+        return User(self._post('users/%s?update_existing=1' % user_id, user.json))
```

Comment:
Two things here: this update endpoint doesn't return anything. Some of them do, some don't. It's weird and inconsistent, but we should try to account for it so we're not returning empty objects.

Second, the `update_existing` URL parameter is on the bulk create request.